### ISSUE TITLE
There was an error loading OptiFabric!

### ISSUE THREAD

**Issue by HsLaranjinho** [Chocohead/OptiFabric#323]
Optifabric could not find the Optifine jar in the mods folder: 
C:\Users\User\AppData\Roaming\.minecraft\mods
am i suposed to download the optifine for forge and put into my mods folder?

**Comment by sp2934598w3048** [comment]
> am i suposed to download the optifine for forge and put into my mods folder?

Yes. Just put the forge OptiFine Jar into the mods folder with the OptiFabric Jar and it should work.

**Comment by barthotymous** [comment]
> > am i suposed to download the optifine for forge and put into my mods folder?
> 
> Yes. Just put the forge OptiFine Jar into the mods folder with the OptiFabric Jar and it should work.

i did this and im still getting the same error message, not sure what the issue is
